### DISCUSSION TITLE
How to prevent an area from repainting?

### DISCUSSION THREAD

**Discussion by qdwang** [emilk/egui#3090] [Q&A]
I need to prevent an area from repainting because it spent a lot of GPU work to paint the custom area.

Is this possible? Maybe caching the pixels is the solution, but how to do it?

Thanks~

**Comment by stonerfish** [comment]
Maybe use a screenshot?
https://docs.rs/eframe/latest/eframe/struct.Frame.html#method.screenshot
https://github.com/emilk/egui/tree/master/examples/screenshot
